### PR TITLE
Add JCB template for snow DA recentering

### PR DESCRIPTION
**Pull request by CoryMartin-NOAA**
Add JCB template for OOPS-based snow/land ensemble recentering utility

### CODE REVIEW

**Comment by CoryMartin-NOAA** [comment]
@danholdaway is there anything else I need to use this template with JCB?

File: algorithm/snow/fv3jedi_land_ensrecenter.yaml.j2 [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+date: "{{ atm_background_time_iso }}"
+variables: [totalSnowDepth]
+increment mask:
+  variable: [slmsk]
+  maxvalue: 0.99 
+deterministic increment:
+  datetime: "{{ atm_background_time_iso }}"
+  filetype: fms restart
+  state variables: [snodl,vtype,slmsk]
+  datapath: ./anl
```
Comment (danholdaway) [inline algorithm/snow/fv3jedi_land_ensrecenter.yaml.j2:22]: If this is a read it might also need a `filename is datetime templated: true` flag

**Comment by danholdaway** [comment]
> @danholdaway is there anything else I need to use this template with JCB?

LGTM. You just need some kind of jcb-base in GDAS to turn G-W flavored templated into JCB ones.

**Comment by CoryMartin-NOAA** [comment]
Thanks @danholdaway I'll slowly but surely figure out JCB haha

**Review by DavidNew-NOAA — APPROVED** [review]

**Pull request merged**